src/compiler/llvm-to-backend/LLVMToBackend.cpp

```diff
@@ -127,6 +127,8 @@ bool LLVMToBackendTranslator::setBuildOption(const std::string &Option, const st
   } else if (Option == "known-group-size-z") {
     KnownGroupSizeZ = std::stoi(Value);
     return true;
+  } else if (Option == "known-local-mem-size") {
+    KnownLocalMemSize = std::stoi(Value);
```

Comment:
If we're adding this option globally, we should probably also use it in `LLVMToSpirvTranslator` which currently already has its own flag exactly for this (which is already used by `ocl_queue` and `ze_queue`)

Reply:
Won't, as this is an "optimization" flag, vs Spir-v actually needs it for correctness.